scripts/similarity_trainer.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+import sys
+
+from sklearn.feature_extraction.text import TfidfVectorizer
+
+from bugbug import similarity
+from bugbug.similarity import (
+    LSISimilarity,
+    NeighborsSimilarity,
+    Word2VecSoftCosSimilarity,
+    Word2VecWmdRelaxSimilarity,
+    Word2VecWmdSimilarity,
+)
+
+
+def parse_args(args):
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--algorithm",
+        help="Similarity algorithm to use",
+        choices=similarity.model_name_to_class.keys(),
+    )
+    parser.add_argument(
+        "--disable-url-cleanup",
+        help="Don't cleanup urls when training the similarity model",
+        dest="cleanup_urls",
+        default=True,
+        action="store_false",
+    )
+    parser.add_argument(
+        "--nltk_tokenizer",
+        help="Use nltk's tokenizer for text preprocessing",
+        dest="nltk_tokenizer",
+        default=False,
+    )
+    return parser.parse_args(args)
+
+
+def main(args):
+    if args.algorithm == "lsi":
+        model_creator = LSISimilarity
```

Comment:
You should be able to get model_creator like you're doing in the other script, using the model_name_to_class dict.

Reply:
(With just an if to check if neighbors_tfidf_bigrams)